bench/bench_test.go

```diff
@@ -97,7 +97,7 @@ var (
 	)
 )
 
-var testSignatureScheme = signSchemes.ByName("Ed448-Dilithium3")
+var testSignatureScheme = signSchemes.ByName("ed25519")
```

Comment:
As a wise person once said: ¿Por qué no los dos?

Reply:
it just needs to match whatever signature scheme we've selected by default with genconfig and used for the docker mixnet.

Reply:
Okay, I'd make it hybird pq by default.